pillar/journal-pr-public.sls

```diff
@@ -4,3 +4,12 @@ elife:
 journal:
     api_url: http://prod--gateway.elife.internal
     api_url_public: https://api.elifesciences.org
+    status_checks:
+        Annotations: ping/annotations
+        Journal CMS: ping/journal-cms
+        Lax: ping/lax
+        Medium: ping/medium
```

Comment:
Digests!

Reply:
We can proactively change it now as there is little difference (I took this from `builder-private` where it is still pointing there)